### DISCUSSION TITLE
confused about vscode / vscodium

### DISCUSSION THREAD

**Discussion by codingbeast** [VSCodium/vscodium#1982] [Q&A] [answered]
I notice that in vs code we have a option to off the all telemetry(you can goto preference > setting > search telemetry
set to off . ). this is option is already available in vscode then why this vscodium exists ?

is there any other hidden telemetry in vs code ?

**Comment by bvwidt** [comment, marked as answer]
Despite that telemetry is disabled in vscodium (and in vscode it's optional) it's a matter of license.

For reference 👉 https://vscodium.com/#why-does-this-exist
